.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "7"
+  - "8"
```

Comment:
necessary for builds to succeed. Yarn doesn't support node@7 evidently

Reply:
10? 🤷‍♂️